rcsb/exdb/tests/testPubChemEtlWrapper.py

```diff
@@ -83,7 +88,7 @@ def testAFromBootstrap(self):
                 ccFileNamePrefix="cc-abbrev",
                 exportPath=self.__dirPath,
                 rebuildChemIndices=True,
-                numProc=4,
+                numProcChemComp=4,
```

Comment:
Should we lower this one too? Or do you want to keep the number of procs used for the test higher intentionally?

Reply:
This particular line is `numProcChemComp`, the one that is defaulting to 8 (was 12), so it's actually a decrease. So I assume that was reduced based on the Azure agent size (need to confirm). But the `numProc=4` further down was left in from my local testing, so I'll change that to 2.

Reply:
Awesome thanks for confirming!